api/v1/checluster_types.go

```diff
@@ -781,3 +781,11 @@ func (c *CheCluster) IsImagePullerImagesEmpty() bool {
 func (c *CheCluster) IsInternalClusterSVCNamesEnabled() bool {
 	return c.Spec.Server.DisableInternalClusterSVCNames == nil || !*c.Spec.Server.DisableInternalClusterSVCNames
 }
+
+func (c *CheCluster) IsOpenShiftOAuthUser() bool {
```

Comment:
AFAIK Che now supports kubeadmin just fine, should not we drop that ability at all?

Reply:
Only when devworkspace is enabled